### DISCUSSION TITLE
【讨论】大师砸了游戏机后，弟子为什么突然开悟？悟了什么？

### DISCUSSION THREAD

**Discussion by yikeke** [yikeke/tao-of-programming#2] [Q&A]
[讨论来源](https://github.com/ruanyf/weekly/issues/1513#issuecomment-732617926)

![image](https://user-images.githubusercontent.com/40977455/100190704-cc23e080-2f29-11eb-8320-e2f726f8a54e.png)

![image](https://user-images.githubusercontent.com/40977455/100190722-d514b200-2f29-11eb-88f6-0f5ff7653942.png)

欢迎继续讨论～

**Comment by kele1997** [comment]
除了踩碎它，还可以火烧、水淹。大师的意思大概是思维不要被限制在规则的框架内，有时候可以跳出来，可能会有不同的发现。

**Comment by cliuxinxin** [comment]
框架给你解决问题的同时，也给你带来了限制，你打碎了框架，也就破除了限制。

**Comment by YANgYYyu** [comment]
不要过于注重语言/框架等，编程的任何目的都是为了解决现实问题，能解决显示问题的编程就是好编程

**Comment by kooksee** [comment]
多思维模型，在找到一种确定的方法时同时想到，还可能还其他种，或者，我想的可能是错误的

**Comment by fyzzy1943** [comment]
悟了道

**Comment by J-F-Liu** [comment]
没了游戏机，就没了输赢。还有一种理解是，拆开游戏机，自己动手创造游戏，从玩游戏的层次上升到编游戏。

**Comment by kooksee** [comment]
输赢相对于规则，没了游戏规则，就没输赢

**Comment by LawlietKira** [comment]
下次玩游戏机时不能让大师看到，不然又要被砸

**Comment by 86648219** [comment]
“夫唯不争，故天下莫能与之争”，这里很多观念是来源于道家经典的。

**Comment by nobody4t** [comment]
玩游戏的时候不叫我。

**Comment by zhanghaofei** [comment]
思维不要被规则限制了

**Comment by boqie** [comment]
有PDF版吗

**Reply by yikeke** [reply]
源码只有 markdown，有需要的话可以找一下 markdown 转 pdf 的工具～
